syntax = "proto3";

option go_package = "pb/game";

message GameNodePlayerInfoPBComp
{
  uint32 player_size = 1;
}
